src/applications/burials/components/NoFormPage.jsx

```diff
@@ -0,0 +1,491 @@
+          'Plot or interment allowance (Check this box if you incurred expensed for the plot to bury the Veteran’s remains.)': formData[
+            'view:claimedBenefits'
+          ]?.plotAllowance
+            ? 'Selected'
+            : 'Not selected',
+          'Transportation expenses (Transportation of the Veteran’s remains from the place of death to the final resting place)': formData[
+            'view:claimedBenefits'
+          ]?.transportation
+            ? formatCurrency(formData['view:claimedBenefits']?.amountIncurred)
+            : 'None',
+        },
+        'Burial allowance': {
+          'Type of burial allowance':
+            burialAllowanceRequest[(formData?.burialAllowanceRequested)],
+          'Did you previously receive a VA burial allowance?': formData[
+            'view:claimedBenefits'
+          ]?.burialAllowance
+            ? 'Yes'
+            : 'No',
+        },
+        'Plot or interment allowance': {
+          'Place of burial or location of deceased Veteran’s remains': formData?.placeOfRemains
+            ? formData?.placeOfRemains
+            : '',
+          'Was the Veteran buried in a state Veterans cemetary?': formData?.stateCemetary
+            ? 'Yes'
+            : 'No',
+          'Did a federal/state government or the Veteran’s employer contribute to the burial? (Not including employer life insurance)': formData?.govtContributions
+            ? formatCurrency(formData?.amountGovtContribution)
+            : 'No',
+        },
+      };
+    case 'additional-information':
+      return {
+        'Claimant contact information': {
+          Address: formData?.claimantAddress
+            ? formatAddress(formData?.claimantAddress)
+            : '',
+          'Email address': formData?.claimantEmail
+            ? formData?.claimantEmail
+            : '',
+          'Phone number': formData?.claimantPhone
+            ? formatPhoneNumber(formData?.claimantPhone)
+            : '',
+        },
+        'Document upload': {
+          'Veterans death certificate':
+            formData?.transportationReceipts?.length > 0
+              ? formData?.transportationReceipts?.slice(0, 1)
+              : '',
+          'Documentation for transportation of the Veteran’s remains or other supporting evidence':
+            formData?.transportationReceipts?.length > 0
+              ? formData?.transportationReceipts?.slice(
+                  1,
+                  formData?.transportationReceipts?.length,
+                )
+              : '',
+        },
+      };
+    default:
+      return {};
+  }
+};
+
+const ArrayComponent = ({ value }) => {
+  return value[0].size
+    ? value.map((name, index) => {
+        return (
+          <div
+            key={index}
+            className="vads-u-margin-top--4 vads-u-background-color--gray-lightest vads-u-padding--2"
+          >
+            <p className="vads-u-margin-top--0 vads-u-margin-bottom--1">
+              <u>
```

Comment:
Any reason why this `<u>` tag is used here? Been a while since I've seen one of these.

Reply:
[Foundation](https://design.va.gov/foundation/) classhooks do seem more appropriate here, especially since this looks like a one-off...
We have `className="vads-u-text-decoration--underline"`, and for the child `<stong>` tag we also have `className="vads-u-font-weight--bold"`.